inc/object.class.php

```diff
@@ -522,6 +522,11 @@ function displayField($canedit, $name, $value, $template, $description = []) {
       if (!empty($searchoption)
          && !in_array($name, self::getFieldsToHide())) {
 
+         if (strpos($description['Field'], 'plugin_genericobject_emptyspace') === 0) {
```

Comment:
I suggest replacing the test on name by following test. To be able to use any name for spacers.

```suggestion
         if (isset($searchoption['input_type']) && 'emptyspace' === $searchoption['input_type']) {
```